assets/tests/components/detours/diversionPage.test.tsx

```diff
@@ -375,14 +375,14 @@ describe("DiversionPage", () => {
     })
 
     expect(
-      screen.getByRole("button", { name: "Review Detour", hidden: true })
+      screen.getByRole("button", { name: "Review", hidden: true })
```

Comment:
oh instead of changing the text on these selectors, can you instead make the change to use `reviewDetourButton` from selectors?

Reply:
- https://github.com/mbta/skate/pull/2849/files#diff-1535a72a38a9500ad346d1b3d9735aff2c788a2d97cfbf49f2aeaac7aefb26cfR1215
- https://github.com/mbta/skate/pull/2849/files#diff-1535a72a38a9500ad346d1b3d9735aff2c788a2d97cfbf49f2aeaac7aefb26cfR385

Reply:
Oh of course!!

Reply:
Ok I've been playing with this a bit, and I see that this particular assertion includes `hidden: true` in the button `get`, and I'm wondering why? What's the risk of excluding that option

Reply:
If the tests pass, I don't _think_ there is a risk?

From the docs

> `hidden`: If `true` includes elements in the query set that are usually excluded from the accessibility tree. `role="none"` or `role="presentation"` are included in either case.

This would mean that if we were hiding the button via the `hidden` prop, then the query would fail, because looking at this from an accessibility perspective, you can't find a element that is hidden. So this is an escape hatch to let you do so.

Reply:
Sorry for the delayed reply! Yeah, I can't get the test to pass for the one with hidden set, and that's partly because the assertion is a `.not.toBeVisible()`, but I also wondered why the `hidden` was included for this test and not the others.